src/test/kotlin/net/maizegenetics/phgv2/cli/ExportHvcfTest.kt

```diff
@@ -78,4 +78,47 @@ class ExportHvcfTest {
 
     }
 
+    @Test
+    fun testCliktParams() {
+
+        // Test missing dbpath parameter
+        val resultMissingDbpath = ExportHvcf().test(
+            "--sample-names Ref -o $outputHvcfDir"
+        )
+        assertEquals(resultMissingDbpath.statusCode, 1)
+        assertEquals(
+            "Usage: export-hvcf [<options>]\n" +
+                    "\n" +
+                    "Error: missing option --dbpath\n",
+            resultMissingDbpath.output
+        )
+
+        // Test missing sample-names parameter
+        val resultMissingSampleNames = ExportHvcf().test(
+            "--dbpath $dbPath -o $outputHvcfDir"
```

Comment:
I would change the dbPath parameter in the command to be dbPath not dbpath.  This way it resolves to db-path which is consistent with the rest of the commands we use.

Reply:
Ok, did that